src/garnet/base/base_lineedit.py

```diff
@@ -0,0 +1,81 @@
+"""BaseLineEdit
+------------
```

Comment:
please create a folder called helpers and inside create another called UIelements or similar; include the base_lineedit.py and base_statusbar.py there.